log/wrapper.go

```diff
@@ -1,8 +1,7 @@
-// Package log provides common logging interface used by other baseplate
-// packages.
 package log
 
 import (
+	"go.uber.org/zap/zapcore"
 	stdlog "log"
 	"testing"
```

Comment:
style nit: Please put third party imports at the bottom, with a blank line separating them from stdlib imports.